modules/kn-service-offline-create.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="creating-an-offline-service_{context}"]
+= Creating a service using offline mode
+
+.Prerequisites
+
+* {ServerlessOperatorName} and Knative Serving are installed on your cluster.
```

Comment:
Since we're in offline mode, this prereq. can be scratched right? Or maybe marked as `(optional)`.

```suggestion
```

Reply:
Nevermind, it's needed later to test out the create from file.

```
$ kn service create -f test/ksvc/event-display.yaml
```